### PR TITLE
TW-651 Refactor unrelated to farms list changes of epic branch

### PR DESCRIPTION
**Pull request by keshan3262**
This PR doesn’t complete the task

### CODE REVIEW

**Review by herkoss — APPROVED** [review]
GJ!

File: src/components/segmented-control/segmented-control.tsx [hunk 1]
```diff
@@ -57,35 +57,53 @@ export const SegmentedControl = <T extends unknown>({
...
 
+  const renderOption = useCallback(
+    (item: T, index: number) => {
+      const disabled = disabledValuesIndices?.includes(index) ?? false;
+
+      return (
+        <TouchableOpacity
+          disabled={disabled}
+          key={index}
+          style={[styles.itemContainer, { width: tileWidth }]}
```
Comment (Boltman92) [inline src/components/segmented-control/segmented-control.tsx:68]: tileWidth means titleWidth?

**Review by Boltman92 — APPROVED** [review]

File: src/components/segmented-control/segmented-control.tsx (hunk 1 above)
Comment (keshan3262) [reply src/components/segmented-control/segmented-control.tsx:68]: No, it's exactly the width of a tile in a switcher

Comment (Boltman92) [reply src/components/segmented-control/segmented-control.tsx:68]: oops, sorry :)

**Pull request merged**